# Request is region of interest that contains unobserved space and the observing
# camera's pose.
# Response is a list of objects to remove so that the ROI can be observed,
# together with a list of expected observation qualities (portion of target region
# that is visible) for removed object.

asp_msgs/BoundingBox roi
geometry_msgs/PoseStamped camera_pose
---
string[] objects_to_remove # Collision Object names
float32[] information_gain
